CMSgov/mint-app
# [MINT-2499, MINT-3008] Model Plan card in model collab area

# MINT-2499, MINT-3008

## Description

* Created `ModelPlanCard` and its styles and added it to Collaboration Area
* Added Model Plan Card translation
* Added `modifiedByUserAccount` and `taskListStatus` to the `GetModelPlan.ts` query
* Added tests and snaps
* Removed "Remove your Model Plan" from task list side nav


## How to test this change

1.  Navigate to collaboration area and note the new model plan card! 
2. Click to go to task list
3. Make updates to one or all task list items
4. Go back to collaboration area and check to see status is updated and the number of sections started is accurate
5. Verify that the "Share" button is now visible and that the "last modified by" copy is there
6. Check that things match figma and is functioning as expected

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
